content/02.main-text.md

```diff
@@ -211,6 +211,23 @@ manubot cite --render --format=markdown \
   pmid:29618526 doi:10.1038/550143a
 ```
 
+### Interactive features and appearance
+
+Manubot comes with several "plugins" that can be included in manuscripts exported as HTML.
+These plugins add special interactive features that enhance the user experience of viewing and reading manuscripts (Figure @fig:plugins).
+For example, with the "tooltips" plugin enabled, when the user hovers over a link to a reference or figure, a preview of that item pops up above the link, along with controls to navigate between other mentions of that item elsewhere in the document.
+The build process can also accommodate different "themes", which change the general aesthetics and appearance of the exported document (for example, from a contemporary sans-serif style to a more traditional serif style).
+The architecture of the plugins and themes is designed to provide authors with enough flexibility to suit their particular needs and preferences.
+
+![
+**Examples of the various Manubot plugins, illustrating their functionality and usefulness.**
+Screenshots taken from existing manuscripts made with Manubot: [Sci-Hub Coverage Study](https://greenelab.github.io/scihub-manuscript/v/fd7acb7ed0108c920da56f84819ce13f02f68aa8/) and [TPOT-DS](https://trang1618.github.io/tpot-ds-ms/), available under the [CC BY 4.0 License](https://creativecommons.org/licenses/by/4.0/ "Creative Commons Attribution 4.0 International License").
+Clarifying markups overlaid in purple.
```

Comment:
I would add `are` to make this a sentence. Same as above, I would add `were` before taken. I think the convention is that the caption title doesn't need to be a sentence, but the remainder should be sentence based. Not a huge deal probably.